### ISSUE TITLE
Fix examples for consumer group reset offset command

### ISSUE THREAD

**Issue by rkpattnaik780** [redhat-developer/app-services-cli#1192]
## Description

Invalid examples for `kafka consumer-group reset-offset`.  `--topic` is a required flag which is missing from most of the examples.

**Version**: 0.32.0

### Steps to reproduce

1. Run `rhoas consumer-group reset-offset -h`
2. This should show the following examples:
```
# reset offset for a consumer group to latest
$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset latest

# reset offset for a consumer group to earliest
$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset earliest

# reset offset for a consumer group to an absolute value
$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset absolute --value 0

# reset offset for a consumer group to a timestamp
$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset timestamp --value "2016-06-23T09:07:21-07:00"

# reset offset for specific partitions in a consumer group
$ rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset latest --topic my-topic --partitions 0,1
```

### Expected vs actual behaviour

Examples should have `--topic` flag.

**Comment by gkpani97** [comment]
Interested

**Comment by rkpattnaik780** [comment]
Fixed by #1207
